api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/repository/AccountsRepository.java

```diff
@@ -87,4 +87,7 @@ Page<AccountDetails> findAll(Pageable pageable,  @Param("searchTerm") String sea
     List<AccountDetails> findByAccountStatusAndPlatform(String accountStatus, String platform);
     List<AccountDetails> findByAccountStatus(String accountStatus);
 
+    @Query("select a from AccountDetails a where a.platform in (select distinct t.dataSourceName from TargetTypes t) and a.accountStatus='configured'")
+    List<AccountDetails> findAllConfiguredAccounts();
```

Comment:
You have not considered azure subscriptions. azure subscriptions are present in a separate table cf_AzureTenantSubscription. From Ui, user will be selecting and passing subscription id and not tenant id for azure. In the case when user passes azure subscription, it will always fail as per the above query.

Make the relevant change in EE also.

Reply:
done